src/components/Inputs/EmailInput/EmailInput.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { TextInput } from '../TextInput'
+import styles from '../TextInput/TextInput.module.scss'
+import errorStyles from '../Errors.module.scss'
+import useErrorMessage from '../../../hooks/useErrorMessage.js'
+import EmailFormatValidator from '../../../validators/EmailValidator'
+
+const PrivateEmailInput = (props) => {
+  const {
+    name,
+    optional,
+    allCaps,
+    labelCopy,
+    validator,
+    formChangeHandler,
+    initialValue,
+    currentValue,
+    currentError,
+    formTouched,
+    placeholder,
+    ...restProps
+  } = props
+
+  const [getError, setError, getFormattedError, validate] = useErrorMessage(
+    validator
+  )
+  const val = currentValue || initialValue
+  const [touched, setTouched] = useState(val ? true : false)
+  const [value, setValue] = useState(val || '')
+
+  const getClasses = () => {
+    return !!getError()
+      ? `BirthdateInput ${styles.TextInput} ${errorStyles.Error}`
+      : `BirthdateInput ${styles.TextInput}`
+  }
+
+  return (
+    <>
+      <TextInput
+        name="email-input"
+        labelCopy={labelCopy}
+        allCaps={allCaps}
+        initialValue={value}
+        optional={optional}
+        type="email"
+        data-tid={restProps['data-tid']}
+        placeholder={placeholder}
+        currentValue={currentValue}
+        validator={EmailFormatValidator}
+        currentError={currentError}
+        formTouched={formTouched}
+        setFieldTouched={restProps.setFieldTouched}
+      />
+      {getError(currentError, formTouched)}
```

Comment:
sorry, this has changed to `touched` ... I did a big refactor this weekend and switched back to field touched (after researching Formik which uses field blurring by default). I left a lot of research, but, it means the fields pass field touched as second arg here.

Reply:
Updated - but am still trying to figure out if we can validation at `onBlur` rather than `onChange`

Reply:
You may need to pull in the latest `master` ... my sense is you don't have my changes maybe??

Reply:
I cleaned up the component quite a bit (I realized that I didn't need a lot of extra stuff that the other components were using, as this is basically a validation wrapper on top of `TextInput`. That being said, even with your changes in https://github.com/getethos/ethos-design-system/pull/94, I'm still seeing validation on `onChange`, rather than `onBlur`

Reply:
I think you may need to pass in `...restProps`